Workday IAM - TUFE and AD Deactivation (#14577)
demisto/content: Packs/Workday/ReleaseNotes/1_2_3.md

#### Integrations
##### Workday IAM
- Enhanced the ***fetch-incidents*** command:
  - Events of TUFE Users (Involuntary Termination) will automatically be dropped, unless a rehire event is detected.
  - Existing User Profiles with a different source of truth will be marked as *Conversion Hires*.

>   - Added a support for the *IAM - AD User Deactivation* incident type.

Reviewer: ```suggestion
  - Added support for the *IAM - AD User Deactivation* incident type.
```

#### Mappers
##### IAM Sync User - Workday

> - Added a mapping to the *IAM - AD User Deactivation* incident type.

Reviewer: ```suggestion
- Added mapping for the *IAM - AD User Deactivation* incident type.
```
